src/main/java/frc/team2412/robot/sensors/AprilTagsProcessor.java

```diff
@@ -49,8 +50,62 @@ public class AprilTagsProcessor {
 					new Rotation3d(0, Units.degreesToRadians(-30), 0));
 
 	// TODO Measure these
-	private static final Vector<N3> STANDARD_DEVS =
-			VecBuilder.fill(0.1, 0.1, Units.degreesToRadians(5));
+	private static final Vector<N3> STANDARD_DEVS = VecBuilder.fill(1, 1, Units.degreesToRadians(30));
+
+	private static PhotonPipelineResult copy(PhotonPipelineResult result) {
+		var copy =
+				new PhotonPipelineResult(
+						result.getLatencyMillis(), result.targets, result.getMultiTagResult());
+		copy.setTimestampSeconds(result.getTimestampSeconds());
+		return copy;
+	}
+
+	private static final double MAX_POSE_AMBIGUITY = 0.1;
+
+	// Both of these are in radians
+	// Negate pitch to go from robot/target to cam pitch to cam to robot/target pitch
+	private static final double EXPECTED_CAM_TO_TARGET_PITCH = -ROBOT_TO_CAM.getRotation().getY();
+	private static final double CAM_TO_TARGET_PITCH_TOLERANCE = 0.1;
+
+	private static boolean hasCorrectPitch(Transform3d camToTarget) {
+		return Math.abs(camToTarget.getRotation().getY() - EXPECTED_CAM_TO_TARGET_PITCH)
```

Comment:
Might be good to log the reported pitch, to debug any errors with tags being incorrectly filtered out/included.